testrpc/wait.go

```diff
@@ -64,6 +64,7 @@ func WaitUntilNoLeader(t *testing.T, rpc rpcFn, dc string, options ...waitOption
 type waitOption struct {
 	Token                  string
 	WaitForAntiEntropySync bool
+	WaitForService         string
```

Comment:
Add ability to wait for a service because services are no longer synced immediately.